scrapper_config.json

```diff
@@ -1,4 +1,19 @@
 {
-    "seed_urls": [],
-    "total_articles_to_find_and_parse": 0
+    "seed_urls": ["https://www.politlinguistika.ru/jour/issue/view/26",
+                 "https://www.politlinguistika.ru/jour/issue/view/25",
+                 "https://www.politlinguistika.ru/jour/issue/view/24",
+                 "https://www.politlinguistika.ru/jour/issue/view/23",
+                 "https://www.politlinguistika.ru/jour/issue/view/22",
+                 "https://www.politlinguistika.ru/jour/issue/view/21",
+                 "https://www.politlinguistika.ru/jour/issue/view/20",
+                 "https://www.politlinguistika.ru/jour/issue/view/19",
+                 "https://www.politlinguistika.ru/jour/issue/view/18",
+                 "https://www.politlinguistika.ru/jour/issue/view/17",
+                 "https://www.politlinguistika.ru/jour/issue/view/16",
+                 "https://www.politlinguistika.ru/jour/issue/view/15",
+                 "https://www.politlinguistika.ru/jour/issue/view/14",
+                 "https://www.politlinguistika.ru/jour/issue/view/13",
+                 "https://www.politlinguistika.ru/jour/issue/view/12",
+                 "https://www.politlinguistika.ru/jour/issue/view/11"],
+    "total_articles_to_find_and_parse": 2
```

Comment:
have you checked locally that you are able to obtain 100 articles?

Reply:
yes, it works